src/Uno.UI/UI/Xaml/Controls/CommandBar/CommandBar.Partial.cs

```diff
@@ -261,6 +261,12 @@ protected override void OnApplyTemplate()
 
 			base.OnApplyTemplate();
 
+			// UNO Specific
```

Comment:
@MartinZikmund Do we need an issue to track down the underlying root cause and remove the Uno-specific code?